doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -135,6 +137,130 @@ The string "ISO8601" as type for attributes and metadata is also supported. The
 
 [Top](#top)
 
+## `dateModified` and `dateCreated` attributes
+
+Section "Attribute names restrictions" establishes that system/builtin attribute names cannot be used as user attribute names. Thus,
+`dateCreated` and `dateModified` (which are system/builtin attributes) cannot be used as user attribute names.
+
+However, due to legacy reasons, Orion doesn't reject attemps of using these attribute with 400 Bad Request errors. **You are
+strongly encouraged to not use dateCreate/dateModified as attribute names** but, if you at the end need to do so, take into account
+the following information.
+
+You can use dateModified/dateCreated attributes at creation/update time, eg:
+
+```
+POST /v2/entities
+
+{
+  "id": "FutureEntity",
+  "type": "T",
+  ...
+  "dateModified": {
+      "type": "DateTime".
+      "value": "2050-01-01T00:00:00Z"
+  }
+}
+```
+
+Retrieval of such attribute using GET API operations will return the value you created/updated. You don't need to use
+`?attrs=dateModified,...` in this case. Eg:
+
+```
+GET /v2/entities/FutureEntity/attrs
+```
+
+```
+{
+  ...
+  "dateModified": {
+      "type": "DateTime".
+      "value": "2050-01-01T00:00:00Z"
+  }
+}
+```
+
+However, when included in notifications, the value is not the one provided by the user (*), but the real
+one corresponding to entity modification/creation stored internally by Orion, eg (assuming the last update
+for that entity was sent at 11:16:05 at May 28th, 2018):
+
+```
+POST /notify
+
+{
+    "data": [
+        {
+            "dateModified": {
+                "metadata": {},
+                "type": "DateTime",
+                "value": "2018-05-28T11:16:05Z"
+            },
+            "id": "FutureEntity",
+            "type": "T"
+        }
+    ],
+    "subscriptionId": "5b082dc2c17960f8773dd74d"
+}
+```
+
+(*) Exception: initial notification uses the value provided by the user. However, this is probably a side
```

Comment:
the initial notification uses

OR

initial notifications use

Reply:
Fixed in 1ab8d0a878a633c982dbe2f016168e73233d3f1b